packages/cozy-client/src/assets/qualifications.json

```diff
@@ -356,6 +361,6 @@
     "degree", "work", "employment", "revenues", "history", "insurance", "drugs",
     "medical_act", "car", "moto", "truck", "boat", "subscription", "buy/sale",
     "house", "compliance", "building", "food", "real_estate", "tax", "insurance",
-    "education", "statement", "course"
+    "education", "statement", "course", "internet", "phone"
```

Comment:
Nit: Having the values sorted and each on one lien is helpful to detect duplicates and to quickly find a value.

Reply:
Right, but a bit tedious to do it in json. But this is another argument to use a CVS -> JSON

Reply:
In all modern editors ;)
<img width="763" alt="Capture d’écran 2020-11-09 à 11 25 17" src="https://user-images.githubusercontent.com/1107936/98529725-442dad80-227e-11eb-8ac8-797941dac514.png">

Reply:
*mind blown*